api/Trace/TextMapPropagator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+use OpenTelemetry\Context\Context;
+
+/**
+ * @see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#textmap-propagator
+ */
+interface TextMapPropagator
```

Comment:
This used to be `TextMapFormatPropagator`. Renamed and update interface based on latest spec.